internal/rbd/rbd_util.go

```diff
@@ -1407,6 +1462,28 @@ func (rv *rbdVolume) cloneRbdImageFromSnapshot(
 		return fmt.Errorf("failed to set image features: %w", err)
 	}
 
+	if rv.StripCount != 0 {
+		logMsg += fmt.Sprintf(", strip count %d, strip unit %d", rv.StripCount, rv.StripUnit)
+		err = options.SetUint64(librbd.RbdImageOptionStripeCount, rv.StripCount)
+		if err != nil {
+			return fmt.Errorf("failed to set strip count: %w", err)
+		}
+		err = options.SetUint64(librbd.RbdImageOptionStripeUnit, rv.StripUnit)
+		if err != nil {
+			return fmt.Errorf("failed to set strip unit: %w", err)
+		}
+	}
+
+	if rv.ObjectSize != 0 {
+		logMsg += fmt.Sprintf(", object size %d", rv.ObjectSize)
+		err = options.SetUint64(librbd.RbdImageOptionOrder, uint64(rv.ObjectSize))
+		if err != nil {
+			return fmt.Errorf("failed to set object size: %w", err)
+		}
+	}
```

Comment:
Can image create/clone options setup code be factor out, so that setting `RbdImageOptionDataPool`, `RbdImageOptionFeatures`, `RbdImageOptionStripeCount`, etc is done in one place?